src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -257,7 +257,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     winrt::hstring MoveFocusArgs::GenerateName() const
     {
         winrt::hstring directionString;
-        switch (_FocusDirection)
+        switch (FocusDirection())

```

Comment:
why did all these have to change?

Reply:
oh, because of the change to optionals i guess?

Reply:
Yup! It was annoying, but definitely for the better.